packages/runtime/src/lib/constants.ts

```diff
@@ -307,6 +378,20 @@ AssetParamMap[6] = { ...AssetParamMap[5] };
 AssetParamMap[7] = { ...AssetParamMap[6] };
 AssetParamMap[8] = { ...AssetParamMap[7] };
 
+export const AssetParamMapIndex = [
```

Comment:
Enum already exists for this. Use that one and remove this one.

Reply:
Array index must be keep since key of this array is a number and key enum is a variable string. But then I will use Enum string in the array